pkg/asset/installconfig/azure/session.go

```diff
@@ -89,7 +89,8 @@ func credentialsFromFileOrUser(cloudEnv *azureenv.Environment) (*Credentials, er
 	os.Setenv(azureAuthEnv, authFilePath)
 	_, err := auth.NewAuthorizerFromFileWithResource(cloudEnv.ResourceManagerEndpoint)
 	if err != nil {
-		logrus.Debug("Could not get an azure authorizer from file. Asking user to provide authentication info")
+		logrus.Debugf("Could not get an azure authorizer from file: %s", err.Error())
+		logrus.Debugf("Asking user to provide authentication info")
```

Comment:
Does it make sense to fallback to asking for auth info if the cert failed? Maybe we could just fail and let the user decide if they want to fix their certs or move over to client secret auth?

Reply:
This is a more general use case where if anything in the service principal is wrong, we default to creating a new one. I'm not sure if we need to change this for just certs use case.